slo-monitor/src/monitors/pod_monitor.go

```diff
@@ -130,7 +130,15 @@ func (pm *PodStartupLatencyDataMonitor) Run(stopCh chan struct{}) error {
 		&cache.ListWatch{
 			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
 				// This call is paginated by cache.Reflector.
-				return pm.kubeClient.CoreV1().Pods(v1.NamespaceAll).List(options)
+				o := metav1.ListOptions{
```

Comment:
Add a comment explaining why we're doing this strange thing (also remove the comment above, it's no longer really relevant).

Reply:
Done.